scripts/services/migrations.ts

```diff
@@ -18,7 +18,7 @@ export const migration: T.ExpectedExports.migration = compat.migrations
               delete config.username;
               await effects.writeFile({
                 path: "start9/password.dat",
-                toWrite: config.password || "",
+                toWrite: "password: "+ config.password || "",
```

Comment:
What's the reason for this change? Seems it makes the file more complex

Reply:
necessary for `yq` to work

Reply:
I could prob just `cat` the file instead in the entry point...